setup/kubernetes/google.md

```diff
@@ -78,6 +78,18 @@ gcloud beta container --project "$PROJECT_ID" \
     --max-nodes "8"
 ```
 
+## Access Control
+
+GKE allows for integration between Identity Access and Management (IAM) and Kubernetes'
+native Role-Based Access Control (RBAC) mechanism to authorize user actions in the
+cluster. IAM configuration is primarily applied at the Google Cloud Project level,
+and all clusters within that Project. Kubernetes RBAC configuration applies to each
+individual cluster, allowing for fine-grained authorization down to the namespace
+level.
+
+For more information, see [Using Kubernetes RBAC auhtorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
```

Comment:
FYI, drive-by commenter :car: auhtorization -> authorization

If you're using VSCode to edit, I've found the [Code Spell Checker extension](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) to be excellent... I should add that to our recommendations in m :thinking:

Reply:
ah good catch - thank you sir